docs/README.md

```diff
@@ -46,7 +46,7 @@
 - object creation patterns tutorial - factory , constructor pattern, prototype pattern
 - JavaScript ES2020
 - Tricky JavaScript interview questions and answers
-
+- DOM Manipulation
```

Comment:
Pls add this topic under the beginner section

Reply:
Done.